src/styles/dropdown.css

```diff
@@ -0,0 +1,40 @@
+.sub-nav li {
+  padding: 18px;
+  width: calc(100%-18px);
```

Comment:
Please add spaces between opening and closing parentheses, and arguments, in a function invocation or control flow expression:
```js
fn(param1, param2) { /* ... */ }
if(expression) { /* ... */ }
```
should be
```js
fn( params1, param2 ) { /* ... */ }
if( expression ) { /* ... */ }
```